### DISCUSSION TITLE
Changing Thonny background color for exam

### DISCUSSION THREAD

**Discussion by vanie76** [thonny/thonny#3349] [General]
Hi
  Is there a way to create a Custom Configuration Files:  Where i set a  preferred settings and share that file with my students for examination to prevent cheating.

**Comment by winsvin** [comment]
Yes! It is important!

**Comment by aivarannamaa** [comment]
You could explore https://github.com/thonny/thonny/wiki/DeploymentOptions for some advanced configuration options.
